### PR TITLE
[CN-1351] Add note about security client configuration

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

**Comment by kutluhanmetin** [comment]
Shall we also add a note to let user know that he can also use a secret for custom config?

**Comment by hasancelik** [comment]
@kutluhanmetin it has been already added by https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/217

File: docs/modules/ROOT/pages/custom-config.adoc [hunk 1]
```diff
@@ -144,3 +145,7 @@ Some Hazelcast features require the management of additional K8s resources (e.g.
 === Networking configuration
 
 The Hazelcast Platform Operator communicates with the Hazelcast cluster, requiring a specific networking configuration. Reconfiguring it using Custom Config might break communication between the Operator and the cluster. To avoid such cases, the Operator will always override the networking section of the Custom Config. If you require any specific networking setup, please use Hazelcast CR.
+
+=== Security configuration
+
+When configuring `security` with enabled `client-authentication`, you must provide a user with `all` permissions and a simple authentication method. The client and the operator will use this user to connect to and manage the cluster.
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/custom-config.adoc:151]: @SeriyBg could you update existing configmap and secret samples based on this requirements?

**Review by hasancelik — CHANGES_REQUESTED** [review]

**Review by kutluhanmetin — APPROVED** [review]

File: docs/modules/ROOT/pages/custom-config.adoc (hunk 1 above)
Comment (SeriyBg) [reply docs/modules/ROOT/pages/custom-config.adoc:151]: We don't have the sample of custom configs in the operator codebase. In the docs, however, there are two examples with the security, but it does not enable client auth.

Comment (hasancelik) [reply docs/modules/ROOT/pages/custom-config.adoc:151]: > In the docs, however, there are two examples with the security, but it does not enable client auth.

I think we should modify them or at least we should another provide sample YAML that contains requires stuff directly here

Comment (SeriyBg) [reply docs/modules/ROOT/pages/custom-config.adoc:157]: Done

**Pull request merged**